en/api/Phalcon_Annotations_Annotation.rst

```diff
@@ -60,13 +60,13 @@ Returns an argument in a specific position
 
 
 
-public *mixed*  **getNamedArgument** (*string* $name)
+public *mixed*  **getNamedArgument** (*unknown* $name)
 
 Returns a named argument
 
 
 
-public *mixed*  **getNamedParameter** (*string* $name)
+public *mixed*  **getNamedParameter** (*unknown* $name)
```

Comment:
https://github.com/phalcon/cphalcon/blob/master/phalcon/annotations/annotation.zep#L206-L214
```zephir
/**
 * Returns a named parameter
 *
 * @return mixed
 */
public function getNamedParameter(string! name)
{
	return this->getNamedArgument(name);
}
```
Removed in phalcon/cphalcon@379e45d.
See also phalcon/cphalcon@af3f7b65 (annotations docblock cleanup).